### PR TITLE
sessions

### PR DESCRIPTION
**Pull request by dzmitrykashlach**

### CODE REVIEW

File: src/main/java/com/blazemeter/api/explorer/Master.java [hunk 1]
```diff
@@ -71,13 +71,12 @@ public String getJUnitReport() throws IOException {
      */
     public List<Session> getSessions() throws IOException {
         logger.info("Get list of sessions for master id=" + getId());
-        String uri = utils.getAddress() + String.format("/api/v4/masters/%s/sessions", encode(getId()));
+        String uri = utils.getAddress() + String.format("/api/v4/masters/sessions?masterId=%s", encode(getId()));
```
Comment (artem-fedorov) [inline src/main/java/com/blazemeter/api/explorer/Master.java]: `/api/v4/sessions?masterId=<master_id>`

File: src/test/java/com/blazemeter/api/explorer/MasterTest.java [hunk 2]
```diff
@@ -415,10 +412,10 @@ public void testPostProperties() {
         Master master = new Master(emul, "id", "name");
         master.postProperties("1=2");
         assertEquals(2, emul.getRequests().size());
-        assertEquals("Request{method=GET, url=http://a.blazemeter.com/api/v4/masters/id/sessions, tag=null}", emul.getRequests().get(0));
+        assertEquals("Request{method=GET, url=http://a.blazemeter.com/api/v4/masters/sessions?masterId=id, tag=null}", emul.getRequests().get(0));
```
Comment (artem-fedorov) [inline src/test/java/com/blazemeter/api/explorer/MasterTest.java]: `/api/v4/sessions?masterId=<master_id>`

File: src/test/java/com/blazemeter/api/explorer/MasterTest.java [hunk 3]
```diff
@@ -121,10 +121,10 @@ public void testGetSessions() throws Exception {
         assertEquals("12", sessionsList.get(0).getUserId());
         assertEquals("13", sessionsList.get(0).getTestId());
         assertEquals("r-v3-1234567890qwerty", sessionsList.get(0).getId());
-        assertEquals("Request{method=GET, url=http://a.blazemeter.com/api/v4/masters/id/sessions, tag=null}", emul.getRequests().get(0));
+        assertEquals("Request{method=GET, url=http://a.blazemeter.com/api/v4/masters/sessions?masterId=id, tag=null}", emul.getRequests().get(0));
```
Comment (artem-fedorov) [inline src/test/java/com/blazemeter/api/explorer/MasterTest.java]: `/api/v4/sessions?masterId=<master_id>`

File: src/test/java/com/blazemeter/api/explorer/MasterTest.java [hunk 4]
```diff
@@ -467,9 +464,9 @@ public void testPostPropertiesFailPostPropToSessions() throws Exception {
 
         String logs = logger.getLogs().toString();
         assertTrue(logs, logs.contains("Failed to send properties for session id=r-v3-1234567890qwerty"));
-        assertEquals(logs, 545, logs.length());
+        assertEquals(logs, 541, logs.length());
 
-        assertEquals("Request{method=GET, url=http://a.blazemeter.com/api/v4/masters/id/sessions, tag=null}", emul.getRequests().get(0));
+        assertEquals("Request{method=GET, url=http://a.blazemeter.com/api/v4/masters/sessions?masterId=id, tag=null}", emul.getRequests().get(0));
```
Comment (artem-fedorov) [inline src/test/java/com/blazemeter/api/explorer/MasterTest.java]: `/api/v4/sessions?masterId=<master_id>`

File: src/test/java/com/blazemeter/ciworkflow/CiBuildTest.java [hunk 5]
```diff
@@ -98,7 +97,7 @@ public void testExecute() throws Exception {
         assertEquals("Request{method=GET, url=http://a.blazemeter.com/api/v4/masters/responseMasterId/status?events=false, tag=null}", emul.getRequests().get(i++));
         assertEquals("Request{method=GET, url=http://a.blazemeter.com/api/v4/masters/responseMasterId/status?events=false, tag=null}", emul.getRequests().get(i++));
         assertEquals("Request{method=PATCH, url=http://a.blazemeter.com/api/v4/masters/responseMasterId, tag=null}", emul.getRequests().get(i++));
-        assertEquals("Request{method=GET, url=http://a.blazemeter.com/api/v4/masters/responseMasterId/sessions, tag=null}", emul.getRequests().get(i++));
+        assertEquals("Request{method=GET, url=http://a.blazemeter.com/api/v4/masters/sessions?masterId=responseMasterId, tag=null}", emul.getRequests().get(i++));
```
Comment (artem-fedorov) [inline src/test/java/com/blazemeter/ciworkflow/CiBuildTest.java]: `/api/v4/sessions?masterId=<master_id>`

**Comment by artem-fedorov** [comment]
@doratias  Please confirm that we should use `/api/v4/sessions?masterId=<master_id>` instead of `/api/v4/masters/%s/sessions`

**Comment by doratias** [comment]
yes @artem-fedorov

File: src/main/java/com/blazemeter/api/explorer/Master.java (hunk 1 above)
Comment (dzmitrykashlach) [reply src/main/java/com/blazemeter/api/explorer/Master.java]: @artem-fedorov sorry for bug :-) fixed.

**Pull request merged**